src/librustc_typeck/diagnostics.rs

```diff
@@ -3284,6 +3284,82 @@ impl Baz for Bar { } // Note: This is OK
 ```
 "##,
 
+E0374: r##"
+The trait `CoerceUnsized` may only be implemented for a coercion between
```

Comment:
Oh sorry. Just a note, I was thinking that you were supposed to put the actual error text before the error description.